features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/source/src/app/components/Apis/Details/Endpoints/GeneralConfiguration/UploadCertificate.jsx

```diff
@@ -0,0 +1,303 @@
+                        }
+
+                        <TextField
+                            required
+                            id='certificateAlias'
+                            label={
+                                <FormattedMessage
+                                    id='Apis.Details.Endpoints.GeneralConfiguration.UploadCertificates.alias'
+                                    defaultMessage='Alias'
+                                />
+                            }
+                            value={alias}
+                            placeholder='My Alias'
+                            onChange={event => setAlias(event.target.value)}
+                            margin='normal'
+                            variant='outlined'
+                            fullWidth
+                        />
+                        <Dropzone
+                            multiple={false}
+                            accept={
+                                'application/pkcs8,' +
+                                    'application/pkcs10, application/pkix-crl,' +
+                                    'application/pkcs7-mime,' +
+                                    'application/x-x509-ca-cert,' +
+                                    'application/x-x509-user-cert,' +
+                                    'application/x-pkcs7-crl,' +
+                                    'application/x-pkcs12,' +
+                                    'application/x-pkcs7-certificates,' +
+                                    'application/x-pkcs7-certreqresp,' +
+                                    '.p8, .p10, .csr, .cer, .crl, .p7c, .crt, .der, .p12, .pfx, .p7b, .spc, .p7r'
+                            }
+                            className={classes.dropzone}
+                            activeClassName={classes.acceptDrop}
+                            rejectClassName={classes.rejectDrop}
+                            onDrop={(dropFile) => {
+                                onDrop(dropFile);
+                            }}
+                        >
+                            {({ getRootProps, getInputProps }) => (
+                                <div {...getRootProps({ style: dropzoneStyles })}>
+                                    <input {...getInputProps()} />
+                                    <div className={classes.dropZoneWrapper}>
+                                        {certificate.name === '' ? (
+                                            <div>
+                                                <Icon style={{ fontSize: 56 }}>cloud_upload</Icon>
+                                                <Typography>
+                                                    <FormattedMessage
+                                                        id={
+                                                            'Apis.Details.Endpoints.GeneralConfiguration' +
+                                                                '.UploadCertificates.click.or.drop.to.upload.file'
```

Comment:
Isn't this Apis.Details.Endpoints.GeneralConfiguration.UploadCertificate.click.or.drop.to.upload.file ?
File name is UploadCertificate

Reply:
Fixed with 3b452e704b181fd0b6a0194f0626481050b1e971